index.js

```diff
@@ -235,7 +242,11 @@ export default class ImageView extends Component<PropsType> {
         this.renderCounter = 1;
 
         // calculate initial scale and translate for images
-        this.imageInitialParams = props.images.map(getInitalParams);
+        let initialScreenDimensions = {
```

Comment:
`const initialScreenDimensions `

Reply:
Moreover this pattern: 
```js
{
  screenWidth: Dimensions.get('window').width,
  screenHeight: Dimensions.get('window').height,
}
``` 
is repeating so its better to place it in function like 

```js

const getScreenDimensions () => ({
  screenWidth: Dimensions.get('window').width,
  screenHeight: Dimensions.get('window').height,
})
```

and use it here and in the beggining